src/server.c

```diff
@@ -1363,6 +1377,7 @@ dictType shaScriptObjectDictType = {
 /* Db->expires */
 dictType keyptrDictType = {
```

Comment:
i think we need to rename this object now that it's no longer a simple "keyptr dict type"
```suggestion
dictType dbExpiresDictType = {
```